### DISCUSSION TITLE
i cannot see my nft on seplia

### DISCUSSION THREAD

**Discussion by AVI-SHUKLA1110** [Cyfrin/foundry-full-course-cu#1139] [Q&A]
hello there ! i minted my first nft on sepolia but i  cannot see it after i imported it on my metamask, here is what it looks like - 
<img width="350" alt="image" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/88367072/7a08bc66-0519-4618-b1a7-cd8583495b80">

**Comment by Niferu** [comment]
Hello @AVI-SHUKLA1110 

Go to meta mask -->> settings -->> advanced -- >> clear meta mask logs.
Then go to security and privacy, enable autoDetectNft and openSea api

**Comment by AVI-SHUKLA1110** [comment]
@Nlferu  i did whatever u suggested me but it didnt came , this is wha t i am getting - 

<img width="257" alt="image" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/88367072/649f9844-05e4-4ef3-a17c-ac718b536f2e">
 BUT  i am getting the nfts only when i am opening these images in new tab , not on the nft page

**Comment by AVI-SHUKLA1110** [comment]
is this happening bcz i m using sepolia or what?

**Reply by Niferu** [reply]
Not it is not network dependant. Recently many people have this bug and it is coming from MetaMask probably. Look here [link](https://github.com/Cyfrin/foundry-full-course-f23/discussions/1054)
